template/functions/curator/fn_curatorVehicleCargo.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+	XPT_fnc_curatorVehicleCargo
+	Author: blah2355
+	Sets custom cargo to Zeus spawned vehicles and box objects.
+	
+	Parameters:
+		0: Object - Vehicle or box spawned by a curator
+		
+	Returns: Nothing
+*/
+
+params [
+	["_curator", nil, [objNull]],
+	["_vehicle", nil, [objNull]]
+];
+
+// Exit if the vehicle is undefined
+if (isNil "_vehicle") exitWith {
+	[2, "Called with no vehicle defined", 0] call XPT_fnc_log;
+};
+
+// Retrieve vehicle data from the config files
+private _itemCargo = [((_class >> "itemCargo") call BIS_fnc_getCfgData)] param [0, nil, [""]];
+
```

Comment:
This would need some sort of check here to make sure that the class of the object exists as an item cargo class before loading `XPT_fnc_loadItemCargo`, otherwise it'll error out.